src/backend/ManagementHub.Service/Areas/Tournaments/TournamentManagerViewModel.cs

```diff
@@ -0,0 +1,8 @@
+namespace ManagementHub.Service.Areas.Tournaments;
+
+public class TournamentManagerViewModel
+{
+	public string? Id { get; set; }
```

Comment:
Id should be of type UserIdentifier here. Moreover, all those fields should be non-nullable.